discord/permission.go

```diff
@@ -109,16 +117,15 @@ const (
 		PermissionMoveMembers |
 		PermissionUseVAD |
 		PermissionPrioritySpeaker |
-		PermissionRequestToSpeak
+		PermissionRequestToSpeak |
+		PermissionStartEmbeddedActivities
```

Comment:
PermissionStartEmbeddedActivities is voice?

Reply:
Yes
> Allows for launching activities (applications with the EMBEDDED flag) in a voice channel
https://discord.com/developers/docs/topics/permissions#permissions-bitwise-permission-flags